index.js

```diff
@@ -404,15 +404,19 @@ export default function(value, nominatim_object, optional_conf_parm) {
      */
     function getRegexKeyForKeyFromOsmDefaults(key) {
         var regex_key;
+        var exact_match = false;
 
-        for (var osm_key in osm_tag_defaults) {
+        Object.keys(osm_tag_defaults).forEach(function (osm_key) {
+            if (exact_match === true) {
+                return;
+            }
             if (key === osm_key) { // Exact match.
                 regex_key = osm_key;
-                break;
```

Comment:
> Ah, I knew I missed something 😄 Sure, the transpile option would be preferred. You can do this when you want as discussed above. If not, maybe add an comment to explain why this was needed, otherwise, it might get refactored since all tests are passing even with the old code. You might want to add an integration test to ensure that compatibility stays and does not get broken again.

Ok I add a comment and I squash. Unfortunately for the integration test... it depends on the js vm (especially the IE one).

About the ES6 rewrite & transpilation I will give a try soon after this PR merge.